Imperium - Grey Knights.cat

```diff
@@ -10999,7 +10999,7 @@
       <rules/>
       <infoLinks/>
       <modifiers/>
-      <description>When a friendly Grey Knights units within 6&quot; of a Brother-Captains manifests the Smite power, double its range.</description>
+      <description>When a friendly Grey Knights units within 6&quot; of a Brother-Captain manifests the Smite power, double its range.</description>
```

Comment:
Brother-Captains is actually how it shows up in the manual, if anything we should put parenthesis around it since it's a category/keyword.